### PR TITLE
dhcp reservation by hostname, cleanup, and remove two sections that a…

### PR DESCRIPTION
**Pull request by julialongtin**
…re now redundant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,33 +191,59 @@ sudo apt dist-upgrade
...
 * As root on each of the nodes, add the following line at the end of the /etc/sudoers file:
 ```
 <ANSIBLE_LOGIN_USERNAME>     ALL=(ALL) NOPASSWD:ALL
 ```
 Replace `<ANSIBLE_LOGIN_USERNAME>` with the username of the account you set up when you installed the machine.
 
+
+#### Fix IPs.
+FIXME: not working?
 It is important that the IPs of machines do not change.
```
Comment (fisx) [inline README.md]: why?

File: README.md [hunk 2]
```diff
@@ -249,42 +257,63 @@ host <hostname> {
...
+  pool {
+    allow members of "admin";
+    range 10.0.0.38 10.0.0.38;
+  }
+}
+```
 
-helm add the wtf repo for kubernetes-charts.storage.googleapis.com
+#### Deploying wire
+From here, follow wire-server-deploy/ansible/README.md , with the following exception:
```
Comment (fisx) [inline README.md]:
```suggestion
From here, follow wire-server-deploy/ansible/README.md, with the following exception:
```

File: README.md (hunk 1 above)
Comment (fisx) [reply README.md]: also, this is why i'm anal about empty lines and whitespace in general: this code adds the second line to the FIXME.

Comment (fisx) [reply README.md]: consistency / greppability: we use `TODO` for `should really be solved`, `FUTUREWORK` for `we would really like to solve it, but it's not technically urgent or even important`.  `FIXME` is rarer.

**Comment by fisx** [comment]
re-approved!

**Comment by julialongtin** [comment]
pending the merge of https://github.com/wireapp/wire-server-deploy/pull/71.

**Comment by fisx** [comment]
> pending the merge of [wireapp/wire-server-deploy#71](https://github.com/wireapp/wire-server-deploy/pull/71).

done :)

**Pull request merged**